resources/project.rb

```diff
@@ -9,13 +9,14 @@
 default_action :install
 
 attribute :project_dir, :kind_of => String, :name_attribute => true
-attribute :vendor, :kind_of => String, :name_attribute => true, :required => true
```

Comment:
Anyone using the vendor attribute on the LWRP will run into issues (BC break). If possible, I'd rather stay away from upping the major version number - any chance we could maintain the vendor attribute (mark it as deprecated when used - display a [warning](https://github.com/djoos-cookbooks/newrelic/blob/master/providers/deployment.rb#L17-L21)) and then just pop it into the package variable?

Let me know what you think!